qutip/solver/mcsolve.py

```diff
@@ -373,10 +367,6 @@ def _do_collapse(self, collapse_time, state):
     def arguments(self, args):
         if args:
             self._integrator.arguments(args)
-            for c_op in self._c_ops:
-                c_op.arguments(args)
-            for n_op in self._n_ops:
-                n_op.arguments(args)
```

Comment:
Why did you remove the update here? The mc integrator is the one using the c_ops and n_ops so it should update them if the arguments are changed. Sure they are a reference to the ones in the MCRHS. But that would be the same for the the `rhs` in the integrator...

Reply:
I had this idea that this should not be the responsibility of the integrator. But you are right, I changed it back. I guess it doesn't hurt to call `arguments` multiple times with the same args.